### ISSUE TITLE
Duplicati.CommandLine.RecoveryTool the tool does not create a hash file

### ISSUE THREAD

**Issue by rdbox** [duplicati/duplicati#3741]
I can't understand what I'm doing.
I see a message that Khash will not find why?

```
0: d:\zzz_recovery\duplicati-20190417T204302Z.dlist.zip - Not a Duplicati file, ignoring
0: d:\zzz_recovery\duplicati-20190417T204350Z.dlist.zip - Not a Duplicati file, ignoring
0: d:\zzz_recovery\duplicati-20190417T204413Z.dlist.zip - Not a Duplicati file, ignoring
0: d:\zzz_recovery\duplicati-20190417T213024Z.dlist.zip - Not a Duplicati file, ignoring
0: d:\zzz_recovery\duplicati-b3d3546f47fe048d884ce079d73a359a5.dblock.zip - Not a Duplicati file, ignoring
0: d:\zzz_recovery\duplicati-b5f99d257d6c645248a258982a00590cd.dblock.zip - Not a Duplicati file, ignoring
0: d:\zzz_recovery\duplicati-be9676abc7e7e4e74845f7e1f295e1f27.dblock.zip - Not a Duplicati file, ignoring
0: d:\zzz_recovery\in.txt - Not a Duplicati file, ignoring
Processed 0 files and found 0 hashes
```

**in console**
```
Duplicati.CommandLine.RecoveryTool.exe index \
"d:\zzz_recovery" \
--indexfile="d:\zzz_recovery\in.txt" \
--full-result=true \
--console-log-level=verbose 
```

**Comment by rdbox** [comment]
[Solution here](https://forum.duplicati.com/t/duplicati-commandline-recoverytool-the-tool-does-not-create-a-hash-file/6864/12)

**Comment by warwickmm** [comment]
Fixed in pull-request #4258.
